docs/apache-airflow-providers-microsoft-psrp/index.rst

```diff
@@ -48,6 +48,7 @@
     :maxdepth: 1
     :caption: Resources
 
+    Example DAGs <https://github.com/apache/airflow/tree/providers-microsoft-psrp/|version|/tests/system/providers/microsoft/winrm>
```

Comment:
```suggestion
    Example DAGs <https://github.com/apache/airflow/tree/providers-microsoft-psrp/|version|/tests/system/providers/microsoft/psrp>
```

Reply:
In additional you need to add this toctree section under the toctree section References (Line 50)

```
.. toctree::
    :hidden:
    :maxdepth: 1
    :caption: System tests

    System Tests <_api/tests/system/providers/microsoft/psrp/index>
```

Reply:
Interesting, I haven't used Sphinx before. Was there any hint here that could have told me that? I did a local docs build but it did not say more than the warning I mentioned earlier.

Reply:
That solved the problem btw! :)